wolfcrypt/src/ASN_TEMPLATE.md

```diff
@@ -0,0 +1,113 @@
+
+
+# Writing a template
```

Comment:
Should this new file `wolfcrypt/src/ASN_TEMPLATE.md` be added to an include.am EXTRA_DIST?

Reply:
Added to EXTRA_DIST.